dart/example/vm_service_lib_tester.dart

```diff
@@ -35,8 +35,10 @@ main(List<String> args) async {
   print('dart process started');
 
   process.exitCode.then((code) => print('vm exited: ${code}'));
-  process.stdout.transform(UTF8.decoder).listen(print);
-  process.stderr.transform(UTF8.decoder).listen(print);
+  // TODO: We shouldn't have to use three of these.
+  // ignore: strong_mode_down_cast_composite
```

Comment:
No. @Hixie was confused by this too as I recall.  If you find it too unnatural as is, feel free to open an issue for EOLs.